privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -186,14 +207,44 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         viewModelScope.launch { updateSite(site) }
     }
 
+    private fun pixelParamList(): List<String> {
+        // we're only running for english speaking users initially.
+        val isEnglish: Boolean = Locale.getDefault().language == Locale.ENGLISH.language
+        if (!isEnglish) return emptyList()
+
+        // if protectionToggleHighlightActive is not enabled, send no params
+        if (!privacyDashboardRemoteFeature.highlightedProtectionsToggle().isEnabled()) return emptyList()
+
+        // otherwise, send the pixel param
+        return listOf(PixelParameter.DASHBOARD_TOGGLE_HIGHLIGHT)
+    }
+
+    private fun pixelParamMap(): Map<String, String> {
+        return pixelParamList().associateWith { true.toString() }
+    }
+
     private suspend fun updateSite(site: Site) {
+        suspend fun createRemoteFeatureSettingsViewState() = withContext(dispatcher.io()) {
+            val layout = if (privacyDashboardRemoteFeature.highlightedProtectionsToggle().isEnabled()) {
```

Comment:
Related to the locale check, if you want to show the new layout to english speaker users, this is a good place to add the check. 

`if (privacyDashboardRemoteFeature.highlightedProtectionsToggle().isEnabled()
            && Locale.getDefault().language == Locale.ENGLISH.language
        )`

If you go for this approach, in the `pixelParamList` method. I believe you can just do: 
```
    private fun pixelParamList(): List<String> {
        val viewState = viewState.value ?: return emptyList()
        if (viewState.remoteFeatureSettings.primaryScreen.layout == LayoutType.DEFAULT.value) return emptyList()
        // otherwise, send the pixel param
        return listOf(DASHBOARD_TOGGLE_HIGHLIGHT)
    }
```